farcy/__init__.py

```diff
@@ -49,6 +49,52 @@
 PR_ISSUE_COMMENT_FORMAT = '_{0}_ {{0}}'.format(MD_VERSION_STR)
 COMMIT_STATUS_FORMAT = '{0} {{0}}'.format(VERSION_STR)
 
+FARCY_COMMENT_START = '_{}_'.format(MD_VERSION_STR)
+
+
+def extract_issues(text):
+    """Extract farcy violations from a text."""
+    if not is_farcy_comment(text):
+        return []
+    # Strip out start of bullet point, ignore first line
+    return [line[2:] for line in text.split('\n')[1:]]
```

Comment:
This could be written as:

    return [line[2:] for line in text.split('\n')[1:]] if is_farcy_comment(text) else []